addons/web/static/src/legacy/js/fields/relational_fields.js

```diff
@@ -2757,6 +2833,70 @@ var FieldMany2ManyTagsAvatar = FieldMany2ManyTags.extend({
     tag_template: 'FieldMany2ManyTagAvatar',
     className: 'o_field_many2manytags avatar',
 
+    on_attach_callback() {
+        const self = this;
+        if (self.viewType === "form") {
+            const provide = async (env, options) => {
+                const many2one = new Many2OneAvatar(self, self.name, self.record, {
+                    mode: 'edit',
+                    noOpen: true,
+                    noCreate: !self.canCreate,
+                    viewType: self.viewType,
+                    attrs: self.attrs,
+                });
+                many2one._getSearchBlacklist = function () {
+                    return self.value.res_ids;
+                };
+                const records = await many2one._search_assign_to(options.searchValue, 10)
+                return records.map((record) => ({
+                    name: record[1],
+                    action: () => {
+                        self._setValue({
+                            operation: 'ADD_M2M',
+                            ids: { id: record[0], display_name: record[1] }
+                        });
+                    },
+                }))
+            };
+            let getCommandDefinition = (env) => ({
+                name: env._t("Assign to ..."),
+                options: { category: "smart_action", hotkey: "alt+i" },
+                action() {
+                    return env.services.command.openPalette({
+                        emptyMessageByNamespace: { "default": env._t("No users found") },
+                        placeholder: env._t("Select a user..."),
+                        providers: [{ provide }],
+                    })
+                },
+            })
+            core.bus.trigger("set_legacy_command", "web.FieldMany2ManyTagsAvatar.assignTo", getCommandDefinition)
+
+            // Command "Assign/unassign to me"
+            getCommandDefinition = (env) => ({
+                name: env._t("Assign/unassign to me"),
+                options: { category: "smart_action", hotkey: "alt+shift+i" },
+                action() {
+                    if (self.value.res_ids.includes(session.user_id[0])) {
+                        self._setValue({
+                            operation: 'DELETE',
+                            ids: [session.user_id[0]],
+                            });
+                    } else {
+                        self._setValue({
+                            operation: 'ADD_M2M',
+                            ids: { id:  session.user_id[0], display_name: session.name }
+                        });
+                    }
+                },
+            });
+            core.bus.trigger("set_legacy_command", "web.Many2OneAvatar.assignToMe", getCommandDefinition);
+        }
+    },
+    on_detach_callback() {
+        core.bus.trigger("remove_legacy_command", "web.FieldMany2ManyTagsAvatar.assignTo");
```

Comment:
Note that for this widget specifically, the relevance of having multiple widgets of that type in one view is very high, so having on the last widget's command active bothers me. I guess we'd need to improve this in the future